static/app/views/issueDetails/streamline/solutionsHubDrawer.spec.tsx

```diff
@@ -37,6 +56,12 @@ describe('AutofixDrawer', () => {
         headline: 'Test headline',
       },
     });
+
+    (useOrganization as jest.Mock).mockReturnValue({
+      ...organization,
+      hideAiFeatures: false,
+      genAIConsent: true,
+    });
```

Comment:
do not mock useOrganization. instead pass your customized organization as a context https://develop.sentry.dev/frontend/using-rtl/#use-our-built-in-contexts